financial-connections/src/test/java/com/stripe/android/financialconnections/features/linkstepupverification/LinkStepUpVerificationViewModelTest.kt

```diff
@@ -109,7 +109,7 @@ class LinkStepUpVerificationViewModelTest {
 
         val viewModel = buildViewModel()
 
-        assertThat(viewModel.awaitState().payload).isInstanceOf(Loading::class.java)
+        assertThat(viewModel.stateFlow.value.payload).isInstanceOf(Loading::class.java)
```

Comment:
Should we test this with Turbine to be more explicit about when we expect a new state and when we don’t?

Reply:
https://github.com/stripe/stripe-android/pull/8155/commits/775c532b5f813f36f90a09848868ead3a5f270b9

Reply:
(Added an example asserting no events are emitted after consumer lookup results in no user found)